motoko/threshold-schnorr/Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: all
+all: build
+
+.PHONY: build
+.SILENT: build
+build:
+	dfx canister create --all
+	dfx build
+
+.PHONY: install
+.SILENT: install
+install: build
+	dfx canister install --all
+
+.PHONY: upgrade
+.SILENT: upgrade
+upgrade: build
+	dfx canister install --all --mode=upgrade
+
+.PHONY: mock
+.SILENT: mock
+mock: install
+	SCHNORR_MOCK_CANISTER_ID=$(shell dfx canister id schnorr_canister); \
+	SCHNORR_EXAMPLE_CANISTER_ID=$(shell dfx canister id schnorr_example_motoko); \
+	echo "Changing to using mock canister instead of management canister for signing"; \
+	CMD="dfx canister call "$${SCHNORR_EXAMPLE_CANISTER_ID}" for_test_only_change_management_canister_id '("\"$${SCHNORR_MOCK_CANISTER_ID}\"")'"; \
```

Comment:
Yes, I tried to do the same but that didn't work with pocket IC because I couldn't figure out a canister id that worked for both (it should be available on a subnet). But since this code is already implemented, it's probably not worth the overhead to change for something that would either way be removed soon.

Reply:
`lqy7q-dh777-77777-aaaaq-cai`, i.e., one of the canister IDs that was assigned automatically when not specifying an ID, worked for me, at least locally. In any case, indeed the existing code works and soon the API will be available from the actual management canister.